_RFCs/0000-template.md

```diff
@@ -0,0 +1,57 @@
+- Change Name: (fill me in with a unique ident, my_awesome_change)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
```

Comment:
do we want a category as well? (standard/informational/feature/etc?)

Reply:
I don't think it's totally necessary as we aren't really sorting these based on category